File: Programmers/level1/문자열다루기기본/seolin.cs
public class Solution {
    public bool solution(string s) {
        bool answer = true;
        int n = 0;
        answer = (s.Length != 4 && s.Length != 6 || int.TryParse(s, out n) == false) ? false : true;
        return answer;
    }
}

File: Programmers/level1/수박수박수박수박수박수/seolin.js
function solution(n) {
  var answer = "";
  for (var i = 0; i < n; i++) answer += i % 2 === 0 ? "수" : "박";
  return answer;
}

//아니 애들 코드 뭔데 초라해지잖아 나
